holoviews/plotting/bokeh/element.py

```diff
@@ -804,15 +804,16 @@ def update_frame(self, key, ranges=None, plot=None, element=None, empty=False):
         # Cache frame object id to skip updating data if unchanged
         previous_id = self.handles.get('previous_id', None)
         if self.batched:
-            current_id = sum(element.traverse(lambda x: id(x.data), [Element]))
+            current_id = sum(element.traverse(lambda x: x._plot_id, [Element]))
         else:
-            current_id = id(element.data)
+            current_id = element._plot_id
```

Comment:
Is it no longer taking into account the ``id`` of the data? It seems to do that in tabular...